lib/api-wmata.js

```diff
@@ -0,0 +1,24 @@
+var request = require('request');
+
+var props = require('../resources/properties');
+var secrets = require('../secret/secrets');
+
+var wmataReq = request.defaults({
+  baseUrl: props.wmata_base_url,
+  qs: secrets.apiKey
+});
+
+var WmataApi = function() {};
+
+WmataApi.prototype.get = function(endpoint, response, callback) {
+  wmataReq(endpoint, function(error, res, body) {
+    if (!error && res.statusCode === 200) {
+      callback(JSON.parse(body));
+    } else {
+      console.error(endpoint.concat(': Error with WMATA'), error);
+      response.tell(props.wmataErrorSpeechOutput);
+    }
+  });
+};
+
+module.exports = new WmataApi();
```

Comment:
If we use `module.exports = wmataGet`, on the other side, we will be saving that function only. There will be no room to pass more data in the exports object. exports is a function at that point.

I like the idea of passing an API key in. In that case, we would not use the `new` keyword in this file, and then anything that uses this would call `new WmataApi(API_KEY)`.

Are we on the same page?